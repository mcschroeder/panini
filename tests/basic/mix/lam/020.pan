f20 : {v:int|?} -> {w:int|?}
f20 = \x:int. 1