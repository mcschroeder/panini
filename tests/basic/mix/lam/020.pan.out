f20 : (v:ℤ) → {w:ℤ | w = 1}